kvstore/badger/go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/pokt-network/smt/kvstore/badger
+
+go 1.20
+
+replace github.com/pokt-network/smt => ../../
```

Comment:
```suggestion
```
Saw your comment, if we remove this, now I have enabled CI for this PR lets see what happens. I can always prepare a follow up PR that is based off this that we ultimately merge in instead

Reply:
I would suggest to remove the line and update to the correct commit in your follow up PR, which should be merged directly after this one.